src/symposium-app/symposium-app.html

```diff
@@ -110,11 +117,11 @@
         </app-toolbar>
 
         <!-- Nav on mobile: side nav menu -->
-        <paper-menu selected="{{selected}}" on-tap="scrollTop">
-            <paper-item>Home </paper-item>
-            <paper-item>Program</paper-item>
-            <paper-item>Location</paper-item>
-            <paper-item>Committee</paper-item>
+        <paper-menu selected="{{data.page}}" attr-for-selected="key" on-tap="scrollTop">
+            <paper-item key="">Home</paper-item>
```

Comment:
Since `key` is not a property of `paper-item`, it is better if you use `data-key` per the HTML spec